migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,84 @@
+
+[discrete]
+[id="mtc-overview-migrate-mtc-applications"]
+== Migrating applications
+You can migrate your applications by using the Migration Toolkit for Containers (MTC) xref:../migration_toolkit_for_containers/migrating-applications-with-mtc.adoc#migrating-applications-mtc-web-console_migrating-applications-with-mtc[web console] or xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migrating-applications-cli_advanced-migration-options-mtc[the command line].
+
+[discrete]
+[id="mtc-overview-advanced-migration-options"]
+== Reviewing advanced migration options
+You can automate your migrations and modify the `MigPlan` and `MigrationController` custom resources in order to perform large-scale migrations and to improve performance. You can check the following items:
+
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-creating-registry-route-for-dim_advanced-migration-options-mtc[Create a registry route for direct image migration]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-configuring-proxies_advanced-migration-options-mtc[Configuring proxies]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-migrating-applications-api_advanced-migration-options-mtc[Migrate an application by using the MTC API]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-state-migration-cli_advanced-migration-options-mtc[Running a state migration]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-hooks_advanced-migration-options-mtc[Creating migration hooks]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-plan-options_advanced-migration-options-mtc[Options for migrating resources]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-controller-options_advanced-migration-options-mtc[Migration controller options for improving performance]
+
+[discrete]
+[id="mtc-overview-troubleshooting-mtc"]
+== Troubleshooting MTC
+You can perform the following troubleshooting tasks:
+
+* xref:../migration_toolkit_for_containers/troubleshooting-mtc.adoc#migration-viewing-migration-plan-resources_troubleshooting-mtc[Viewing plan resources]
+* xref:../migration_toolkit_for_containers/troubleshooting-mtc.adoc#migration-viewing-migration-plan-log_troubleshooting-mtc[Viewing the migration plan aggregated log file]
+* xref:../migration_toolkit_for_containers/troubleshooting-mtc.adoc#migration-using-mig-log-reader_troubleshooting-mtc[Using the migration log reader]
+* xref:../migration_toolkit_for_containers/troubleshooting-mtc.adoc#migration-accessing-performance-metrics_troubleshooting-mtc[Access performance metrics]
```

Comment:
"Access" is "Accessing" in the other PR. Once you finish w/ the wording in the other one, I'd just come back to this one to make sure that any of those updates are made here as well.